### ISSUE TITLE
Adding brand new image

### ISSUE THREAD

**Issue by vallard** [CiscoUcs/KUBAM-Frontend#56]
If the KUBAM file does not exist on the server and the GUI tries to make a new ISO mapping the operation will fail.  It does not fail in the API.  There is something the GUI is doing that creates an invalid map when it tries to do a post.  Subsequent operations work fine.

**Comment by vallard** [comment]
The SSH keys has the same issue.  Perhaps the GUI expects something different when there are no values.  The API works fine.

**Comment by rushins** [comment]
hi 

i tried to map ISO from the KUBAM web interface but i cannot, .i have downloade the ISO on KUBAM docker container inside but nothing is possible to map > 

does anyone know how to map ISO (centos or redhat ) on operating system - ISO images on kubam interface using?

**Comment by vallard** [comment]
is the ISO file inside the ~/kubam/ directory ?

**Comment by rushins** [comment]
Yes it is also i tried mount the iso but getting loop error which I cannot mount ?

**Comment by vallard** [comment]
hmm.. .sorry you are having so much trouble.  Can you get the kubam/kubam
logs?

Run this in one shell:
docker logs -f $(docker ps | awk '/kubam\/kubam/ {print $1}')

and then do the GUI and see if you get any output errors that can help
troubleshoot.

**Comment by vallard** [comment]
Also if you try restarting the container I’ve seen that solve it sometimes as well. We have a bug there.

**Comment by rushins** [comment]
yes after container restart , i see that i can MAP ISO from KUBAM with Centos 7.3. This is fixed but after that i have another issues during Deploy tab.

I have selected Big Blue Button "Make Boot Images" i get this error "OS could not be determined with ISO image. Perhaps this is not a supported OS?" i have downloaded from CENTOS bulids 
http://mirror.ash.fastserv.com/centos/7/isos/x86_64/ and added during Map ISO with Centos 7.3 ? 

any idea what might be the issue ?

**Comment by vallard** [comment]
Looks like a new version of Centos (Centos 7.5).  I'll add support to
that.  Currently we only support Centos 7.2/.3/.4
